# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ploticus/X11R7.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- pl232src/src/Makefile.vanilla	2005-10-09 12:15:40.000000000 +0200
+++ pl232src/src/Makefile	2005-10-09 12:16:27.000000000 +0200
@@ -42,10 +42,10 @@
 #############################################################
 
 #### For Linux uncomment the following.. #######################
-XLIBS = -L/usr/X11R6/lib -lX11
-INCLUDEDIRS = -I/usr/X11R6/include
+XLIBS = -L/usr/X11/lib -lX11
+INCLUDEDIRS = -I/usr/X11/include
 
-#### Note: for 64 bit platform I used: XLIBS = -L/usr/X11R6/lib64
+#### Note: for 64 bit platform I used: XLIBS = -L/usr/X11/lib64
 
 
 #### For Solaris 2.x uncomment the following.. #################
